tests/test_middleware.py

```diff
@@ -14,16 +34,30 @@ def test_init(self):
         # THEN
         assert middleware.get_response == "response"
 
-    def test_add_query_strings_to_links_is_called(self, mocker, request_builder):
+    def test_add_query_strings_to_links_is_called_for_html(self, mocker, request):
         # GIVEN
         mock_add_query_strings_to_links = mocker.patch(
             "global_query_strings.middleware.add_query_strings_to_links"
         )
-        request = request_builder.get
+        middleware = GlobalQueryStringsMiddleware(dummy_html_middleware)
+        assert mock_add_query_strings_to_links.call_count == 0
 
         # WHEN
-        middleware = GlobalQueryStringsMiddleware(request)
         response = middleware(request)
 
         # THEN
         assert mock_add_query_strings_to_links.call_count == 1
+
+    def test_add_query_strings_to_links_is_not_called_for_pdf(self, mocker, request):
+        # GIVEN
+        mock_add_query_strings_to_links = mocker.patch(
+            "global_query_strings.middleware.add_query_strings_to_links"
+        )
+        middleware = GlobalQueryStringsMiddleware(dummy_pdf_middleware)
+        assert mock_add_query_strings_to_links.call_count == 0
+
+        # WHEN
+        response = middleware(request)
```

Comment:
You don't need to set the result in the `response` variable here. Same for  `test_add_query_strings_to_links_is_called_for_html`

Reply:
fixed in 96de5d5